tests/console/sle2docker.pm

```diff
@@ -21,8 +21,11 @@ use utils;
 use registration "install_docker_when_needed";
 use strict;
 use warnings;
+use version_utils 'is_sle';
 
 sub run {
+    return record_soft_failure('bsc#1123502 - missing Container module on SLE12-SP5') if is_sle('=12-sp5');
```

Comment:
Maybe it would be a good idea to only raise the soft failure if the container module is missing.

Something like
```perl
if (is_sle('=12-sp5') && script_run('SUSEConnect --list-extensions | grep "Containers Module"') != 0) {
    return record_soft_failure('bsc#1123502 - missing Container module on SLE12-SP5');
}
```

Reply:
Interesting idea, but not very comfortable with it. We can easily forget about it and it will stay there forever.

Reply:
Which would IMHO be better than forgetting the explicit soft failure as that one reduces the test coverage.

Reply:
No,  it will only show that module is available, not that it is activated. So you have another opportunity for failure when is module added back. Test coverage is already decreased, not sure how can we go even lower. I prefer to fix/update test under control after is module added back to reflect real status of SLE12-SP5, because I don't know at the moment how it will look like after mentioned bsc is fixed.

Reply:
@asdil12 As we talked on IRC, it is not necessary to update PR and it is fine in current state.